.github/workflows/test-application.yaml

```
@@ -193,3 +193,133 @@ jobs:
+        runs-on: windows-latest
+
+        env:
+            JACKRABBIT_VERSION: 2.20.2
+            APP_ENV: test
+            APP_SECRET: a448d1dfcaa563fce56c2fd9981f662b
+            MAILER_URL: null://localhost
+            SULU_ADMIN_EMAIL:
+            SYMFONY_DEPRECATIONS_HELPER: weak
+            PHPCR_TRANSPORT: jackrabbit
+            DATABASE_URL: mysql://root:root@127.0.0.1:3306/sulu_test?serverVersion=${{ matrix.mysql-version }}
+            DATABASE_CHARSET: utf8mb4
+            DATABASE_COLLATE: utf8mb4_unicode_ci
+
+        strategy:
+            fail-fast: false
+            matrix:
+                include:
+                    - php-version: '7.4'
+                      mysql-version: '8.0'
+                      phpcr-transport: 'jackrabbit'
+                      dependency-versions: 'highest'
+                      php-extensions: 'ctype, iconv, intl, mysql, pdo_mysql, php_fileinfo, imagick'
+                      tools: 'composer'
+
+        steps:
+            - name: Checkout project
+              uses: actions/checkout@v2
+
+            - name: Install and configure Node.js
+              uses: actions/setup-node@v2
+              with:
+                  node-version: '14'
+
+            - name: Install and configure PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: ${{ matrix.php-version }}
+                  extensions: ${{ matrix.php-extensions }}
+                  tools: ${{ matrix.tools }}
+                  ini-values: 'memory_limit=-1'
+
+            - name: Install and configure MySQL
+              uses: shogo82148/actions-setup-mysql@v1.0.1
+              with:
+                  mysql-version: ${{ matrix.mysql-version }}
+                  root-password: 'root'
+                  my-cnf: |
+                      [mysqld]
+                      default-authentication-plugin=mysql_native_password
+
+            - name: Cache jackrabbit runnable
+              uses: actions/cache@v2
+              id: jackrabbit-cache
+              if: ${{ matrix.phpcr-transport == 'jackrabbit' }}
+              with:
+                  path: downloads
+                  key: ${{ runner.os }}-jackrabbit-${{ env.JACKRABBIT_VERSION }}
+
+            - name: Download jackrabbit
+              if: ${{ matrix.phpcr-transport == 'jackrabbit' && steps.jackrabbit-cache.outputs.cache-hit != 'true' }}
+              run: |
+                  mkdir downloads
+                  cd downloads
+                  curl -O http://archive.apache.org/dist/jackrabbit/${{ env.JACKRABBIT_VERSION }}/jackrabbit-standalone-${{ env.JACKRABBIT_VERSION }}.jar
+                  cd -
+
+            - name: Run jackrabbit
+              if: ${{ matrix.phpcr-transport == 'jackrabbit' }}
+              run: powershell -command "Start-Process -FilePath java -ArgumentList '-jar downloads/jackrabbit-standalone-${{ env.JACKRABBIT_VERSION }}.jar'"
```

trying to simplify this a bit. see https://github.com/sulu/sulu/pull/5865 🙂 

```suggestion
            - name: Download jackrabbit
              if: ${{ matrix.phpcr-transport == 'jackrabbit' }}
              uses: carlosperate/download-file-action@v1.0.3
              with:
                  file-url: http://archive.apache.org/dist/jackrabbit/${{ env.JACKRABBIT_VERSION }}/jackrabbit-standalone-${{ env.JACKRABBIT_VERSION }}.jar
                  file-name: jackrabbit-standalone.jar
                  location: downloads

            - name: Run jackrabbit
              if: ${{ matrix.phpcr-transport == 'jackrabbit' }}
              run: powershell -command "Start-Process -FilePath java -ArgumentList '-jar downloads/jackrabbit-standalone.jar'"
```